src/test/resources/employee_curd_operation.feature

```diff
@@ -0,0 +1,29 @@
+Feature: employee_curd_operation
+	Employee CRUD operation
+	
+	Scenario: Save employee record
+		Given passing valid employee record
+		When  employee should successfully saved to db
+		Then  it should return saved employee object
+
+	Scenario: is to Get employee by id
+		When  employee id is 36
+		Then  it should return the employee record
+
+	Scenario: is to Get employee by id which is not present in db
+		When  passing invalid employee id
+		Then  it should return empty employee record
+
+	Scenario: is to get all employee
+		Then it should return all the employees
+
+	Scenario: is to update employee by id
+		When passing latest employee with valid id
+		Then it should update with latest employee record for same id
+
+#	Scenario: is to delete existing employee
```

Comment:
Remove

Reply:
resolved as part of below commit 

https://github.com/LokeshReddyExamples/springBootAssignment/pull/4/commits/8a6230e2a67b3dd419b3c263d7c7aa9a9e44c5d6